scripts/backup/periodic_incremental_backup/take_incremental_backup.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+config_file="$1"
+cluster_id="$2"
+
+CURRENT_DIR=$(cd $(dirname $0)  && cd .. && pwd)
+source ${CURRENT_DIR}/util/helper.sh
+
+load_config_file ${config_file}
+set_default_parameters
+check_config_file
+
+#read latest snapshot id
+snapshot_id=$(grpcurl -plaintext -d '{"limit": 1, "cluster_id": "'"$cluster_id"'"}' localhost:20051  rpc.Cassy.ListBackups | jq -r '.entries[0].snapshot_id')
```

Comment:
* What if the latest snapshot is failed ?
* What if it is called during the snapshot is under taken ?

Also, getting the latest snapshot is error-prone.
When a snapshot is successfully taken, it should write the snapshot_id to somewhere.